README.md

```diff
@@ -103,8 +111,8 @@ If everything executes correctly, it should create a file .python.env in the .vs
 From within this repository, install your extension as a Python package to the Isaac Sim Python executable.
 
 ```bash
-${ISAACSIM_PATH}/python.sh -m pip install --upgrade pip
-${ISAACSIM_PATH}/python.sh -m pip install -e .
+${ORBIT_PATH}/orbit.sh -p -m pip install --upgrade pip
+${ORBIT_PATH}/orbit.sh -p -m pip install -e .
```

Comment:
Hmm I would add instructions that if you are in conda you can directly use python. They don't need to worry about orbit.sh at all in the conda.

Reply:
Gotcha, this is done in the reviewed commits.